### PR TITLE
Added version specs on 1.10 upgrading pages and  1.10.5 RN.

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/COPS-2961

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by anithamuthu79** [comment]
@edgarlanting @vinodkone this PR is ready for review.

File: pages/1.10/installing/ent/upgrading/index.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ Proceed with upgrading every master node one-at-a-time in any order using the fo
     1.  Wait until the `dcos-mesos-master` unit is up and running.
     1.  Verify that `curl http://<dcos_master_private_ip>:5050/metrics/snapshot` has the metric `registrar/log/recovered` with a value of `1`.
         **Tip:** If you are upgrading from permissive to strict mode, this URL will be `curl https://...` and you will need a JWT for access.
-    1.  Verify that `/opt/mesosphere/bin/mesos-master --version` indicates that the upgraded master is running Mesos 1.4.0.
+    1.  Verify that `/opt/mesosphere/mesos 1.4.2` indicates that the upgraded master is running Mesos 1.4.2.
```
Comment (vinodkone) [inline pages/1.10/installing/ent/upgrading/index.md]: I don't think `/opt/mesosphere/mesos 1.4.2` is a thing? The previous command was correct. Why did you change the command?

File: pages/1.10/release-notes/1.10.5/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ DC/OS 1.10.5 was released on February 22, 2018.
 # Notable Changes in DC/OS 1.10.5
 
 - Updated to Marathon 1.5.6 ([changelog](https://github.com/mesosphere/marathon/releases/tag/v1.5.6)).
-- Updated to Mesos 1.4.1 ([changelog](https://github.com/apache/mesos/blob/1.4.x/CHANGELOG)).
+- Updated to Mesos 1.4.2-dev ([changelog](https://github.com/apache/mesos/blob/1.4.x/CHANGELOG)).
```
Comment (vinodkone) [inline pages/1.10/release-notes/1.10.5/index.md]: Let's point to the exact sha in the URL: https://github.com/apache/mesos/blob/92d988ce0192ef8cc8bc75575c3a7d18c30ab7ea/CHANGELOG

File: pages/1.11/installing/production/upgrading/index.md [hunk 3]
```diff
@@ -229,7 +229,7 @@ Proceed with upgrading every master node one at a time in any order using the fo
     1.  Verify that `curl http://<dcos_master_private_ip>:5050/metrics/snapshot` has the metric `registrar/log/recovered` with a value of `1`. [enterprise type="inline" size="small" /]
 
         **Note:** If you are upgrading from permissive to strict mode, this URL will be `curl https://...` and you will need a JWT for access. [enterprise type="inline" size="small" /]
-    1.  Verify that `/opt/mesosphere/bin/mesos-master --version` indicates that the upgraded master is running Mesos 1.4.0. [enterprise type="inline" size="small" /]
+    1.  Verify that `/opt/mesosphere/mesos 1.4.2` indicates that the upgraded master is running Mesos 1.4.2. [enterprise type="inline" size="small" /]
```
Comment (vinodkone) [inline pages/1.11/installing/production/upgrading/index.md]: see above.
